app/lib/pre_assembly/batch.rb

```diff
@@ -23,11 +23,12 @@ class Batch
              :set_druid_id,
              :stageable_discovery,
              :staging_style_symlink,
+             :using_file_manifest,
              to: :batch_context
 
     def initialize(batch_context)
       @batch_context = batch_context
-      self.media_manifest = PreAssembly::Media.new(csv_filename: batch_context.media_manifest, bundle_dir: bundle_dir) if batch_context.media_cm_style?
+      self.file_manifest = PreAssembly::FileManifest.new(csv_filename: batch_context.file_manifest, bundle_dir: bundle_dir) if batch_context.using_file_manifest
```

Comment:
Should the `using_file_manifest` be `using_file_manifest?` to conform to normal styles? No strong feelings here, just an observation.

Reply:
Oh maybe not, as it's an initialization parameter as well (below)...

Reply:
yeah, that's a model attribute being passed along, we could do a helper method to alias a `?` style method name

Reply:
I don't think that's necessary, it may make the readability a bit confusing.